frontend/src/components/HospitalMap.tsx

```diff
@@ -17,7 +17,27 @@ class HospitalMap extends React.Component {
     lng: -0.09,
     zoom: 13,
   }
-
+  componentDidMount() {
+    /**
+     * Issue: #12
+     * Title: Detect user's location and use it as a search parameter
+     */
+    if('geolocation' in navigator){ /*Check if browser supports Geolocation API */
+      navigator.geolocation.getCurrentPosition((position) => {
+        console.log(position.coords)
+        /**
+         * Update state with latest Latitude and longitudes
+         */
+        this.setState({
+          lat:position.coords.latitude,
```

Comment:
The position itself is only used after being converted to a `L.latLng` object in the `render` method. It may be simpler if we just stored the `L.latLng` object in the state instead of creating a redundant object. `State` would then just consist of `position` and `zoom` fields.

Reply:
got it

Reply:
Are we not going to use latlng coordinates in some other place? just a thought